docs/Tutorials/EventsAndTriggers.md

```diff
@@ -0,0 +1,72 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+
+# Events and triggers {#tutorial_events_and_triggers}
+
+The "events and triggers" infrastructure (and the related "events and
+dense triggers") provide some control of SpECTRE executable execution
+from the input file contents.  It does not generally have major
+effects on the flow of the algorithm, but is primarily used for
+observations and minor adjustments.
+
+### What is an event?
+
+An \ref Event "event" is an input-file-creatable object that performs
+some task, frequently an observation.  With the exception of the \ref
+Events::Completion "Completion" event, they cannot have direct effects
+on the algorithm, although they can have indirect effects through
+sending messages.  (An example of such an event is \ref
+Events::ChangeSlabSize "ChangeSlabSize".)  In most cases, however, their
+effects on program execution are ephemeral, causing some data to be
+written to a file or standard output but not changing anything about
+the simulation state.
+
+### What is a trigger?
+
+A \ref Trigger "trigger" is an input-file-creatable object that
+controls when events are run.  They are checked periodically, once per
+slab in an evolution executable, and once per iteration in an elliptic
+solve.  At each check, a trigger may fire, causing its associated
+events to run, or not.  Triggers must always be written to be
+independent of element-specific state such as the local values of the
+system variables, so that a trigger will give a consistent result over
+the entire domain at each check.
```

Comment:
This sentence was a bit wordy and confusing, specifically the phrasing of "independent of element-specific state".